api/organisations/subscription_info_cache.py

```diff
@@ -6,16 +6,16 @@
 
 from .chargebee import get_subscription_metadata
 from .models import Organisation, OrganisationSubscriptionInformationCache
-from .subscriptions.constants import CHARGEBEE
+from .subscriptions.constants import CHARGEBEE, SubscriptionCacheEntity
 
 OrganisationSubscriptionInformationCacheDict = typing.Dict[
     int, OrganisationSubscriptionInformationCache
 ]
 
 
-def update_caches():
+def update_caches(update_cache_entity: SubscriptionCacheEntity):
```

Comment:
Let's just use INFLUX and CHARGEBEE, not INFLUX_AND_CHARGEBEE and pass a tuple here.

Reply:
Done.